src/cmake/windows.cmake

```diff
@@ -9,6 +9,13 @@ set_target_properties(mozillavpn PROPERTIES
     VERSION ${CMAKE_PROJECT_VERSION}
     WIN32_EXECUTABLE ON
 )
+# Todo: This will force the generation of a .pdb
```

Comment:
This should use a generator statement so that it's only takes effect when using the Release configuration, otherwise this may interfere with an attempt to create a real debug build.

IIRC that should look something like: `target_compile_options(mozillavpn PRIVATE $<$<CONFIG:Release>:/ZI>>`